figures/montages/scripts/temporal_montages.py

```diff
@@ -0,0 +1,573 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # This PR plots montages of temporal data for a single cell followed through time.
+
+# ## Imports
+
+# In[1]:
+
+
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import PIL
+import tifffile
+from PIL import Image, ImageDraw, ImageEnhance, ImageFont
+
+# ## Functions
+
+# In[2]:
+
+
+def normalize_image(image: np.ndarray) -> np.ndarray:
+    """
+    Normalize the image to the range [0, 255] and convert to uint8.
+
+    Parameters
+    ----------
+    image : np.ndarray
+        Input image to be normalized.
+
+    Returns
+    -------
+    np.ndarray
+        Normalized image in uint8 format.
+    """
+    image = image.astype(np.float32)
+    image -= image.min()
+    image /= image.max()
+    image *= 255.0
+    return image.astype(np.uint8)
+
+
+# make the composite images
+def make_composite_image(
+    image1_path: pathlib.Path,  # yellow
+    image2_path: pathlib.Path,  # green
+    image3_path: pathlib.Path,  # red
+    image4_path: pathlib.Path,  # blue
+) -> PIL.Image.Image:
+    """
+    Create a composite image from four input images.
+
+    Parameters
+    ----------
+    image1_path : pathlib.Path
+        Path to the first image (488_1).
+    image2_path : pathlib.Path
+        Path to the second image (488_2).
+    image3_path : pathlib.Path
+        Path to the third image (561).
+    image4_path : pathlib.Path
+        Path to the fourth image (DNA).
+
+    Returns
+    -------
+    PIL.Image.Image
+        Composite image in CYMK format. Where the channels are:
+        - C: DNA (blue)
+        - M: 488_1 (magenta)
+        - Y: 561 (yellow)
+        - K: 488_2 (green)
+    """
+    # Load the images
+    image1 = tifffile.imread(image1_path)
+    image2 = tifffile.imread(image2_path)
+    image3 = tifffile.imread(image3_path)
+    image4 = tifffile.imread(image4_path)
+
+    # Normalize the images to the range [0, 255]
+    image1 = normalize_image(image1)  # 488_1
+    image2 = normalize_image(image2)  # 488_2
+    image3 = normalize_image(image3)  # 561
+    image4 = normalize_image(image4)  # DNA
+    # make a cyan, magenta, yellow composite
+    # cyan = green + blue
+    # magenta = red + blue
+    # yellow = red + green
+    # composite = max(cyan, magenta, yellow)
```

Comment:
Why did you choose not to include green (488_2) in the end? Why not composite 488_1 and 488_2 into one channel (yellow)? It might be good to have an explanation for that or maybe not load in image2 into the function if it is not used. Just some considerations :D